socorro/processor/mozilla_transform_rules.py

```diff
@@ -578,42 +574,28 @@ class BetaVersionRule(Rule):
     #: If we know it's good, cache it for 24 hours because it won't change
     LONG_CACHE_TTL = 60 * 60 * 24
 
-    #: Products that are on Buildhub--we don't want to ask Buildhub about others
-    BUILDHUB_PRODUCTS = ['firefox', 'fennec', 'fennecandroid']
+    #: List of products to do lookups for
+    SUPPORTED_PRODUCTS = ['firefox', 'fennec', 'fennecandroid']
 
     def __init__(self, config):
         super(BetaVersionRule, self).__init__(config)
-        # NOTE(willkg): These config values come from Processor2015 instance.
-        self.buildhub_api = config.buildhub_api
         self.cache = ExpiringCache(max_size=self.CACHE_MAX_SIZE, default_ttl=self.SHORT_CACHE_TTL)
         self.metrics = markus.get_metrics('processor.betaversionrule')
 
         # NOTE(willkg): These config values come from Processor2015 instance and are
         # used for lookup in product_versions
-        database = config.database_class(config)
-        self.transaction = config.transaction_executor_class(config, database)
+        self.conn_context = config.database_class(config)
```

Comment:
I took out the transaction executor so this rule doesn't need it. Soon I'll be nixing the transaction executor for all things. Figured I'd do it here and see how it went.